tests/src/test/java/com/cloudera/director/google/GoogleLauncherTest.java

```diff
@@ -59,6 +61,8 @@
 
   @BeforeClass
   public static void beforeClass() throws IOException {
+    Assume.assumeFalse(System.getProperty("GCP_PROJECT_ID", "").isEmpty());
```

Comment:
Can this `assumeFalse()` be moved to `testLauncher()`? Then the `testLauncherConfig()` test would still run (I think it does not require the system property).

We could also (as part of a separate PR) work on getting the tests to run live against a dedicated GCP project.